grails-test-suite-uber/src/test/groovy/grails/test/mixin/InheritanceWithValidationTests.groovy

```diff
@@ -47,12 +49,12 @@ class AbstractCustomPropertyValue {
 
 @Entity
 class CustomProperty {
-    AbstractCustomPropertyValue newValue (String value) {
-        return new StringPropertyValue (value)
+    AbstractCustomPropertyValue newValue(String value) {
+        return new StringPropertyValue(value)
     }
 }
 
-@Entity
+//@Entity
```

Comment:
why this ?

Reply:
The test won't compile with the `@Entity` annotation present when extending another domain class.

We could also do this:
```groovy
// With Groovy 4, it is currently not possible to extend domain classes: https://issues.apache.org/jira/browse/GROOVY-5106
@Entity
class StringPropertyValue /*extends AbstractCustomPropertyValue*/ {

    String stringValue

    static constraints = {
        stringValue (nullable: true)
    }

    StringPropertyValue (String value) {
        this.stringValue = value
        this.valid = true
    }
}

```